radicale/config.py

```diff
@@ -277,13 +277,15 @@ def load(paths: Optional[Iterable[Tuple[str, bool]]] = None
         parser = RawConfigParser()
         config_source = "config file %r" % path
         try:
-            if not parser.read(path):
-                config = Configuration.SOURCE_MISSING
-                if not ignore_if_missing:
-                    raise RuntimeError("No such file: %r" % path)
-            else:
+            with open(path, 'r') as f:
+                parser.read_file(f)
                 config = {s: {o: parser[s][o] for o in parser.options(s)}
-                          for s in parser.sections()}
+                        for s in parser.sections()}
+        except FileNotFoundError as e:
```

Comment:
I propose something like the following to avoid the duplicate:
```python
try:
    …
except Exception as e:
    if not (ignore_if_mssing and isinstance(e, FileNotFoundError)):
        raise RuntimeError(…)
    config = Configuration.SOURCE_MISSING
```

Reply:
thanks, fixed.